dist/rpm/openQA.spec

```diff
@@ -566,6 +566,8 @@ fi
 %{_unitdir}/openqa-worker-cacheservice.service
 %{_unitdir}/openqa-worker-no-cleanup@.service
 %{_unitdir}/openqa-worker-auto-restart@.service
+%{_unitdir}/openqa-soft-restart-worker@.service
```

Comment:
not sure how well defined a "soft-restart" is. Maybe just "reload" to explain that the complete worker service is sustained but with new config?

Reply:
Makes sense. I renamed it to `openqa-reload-worker-auto-restart@.path`. I added the `auto-restart` part to make it clear that this unit only affects the `…-worker-auto-restart@.service` but also added a note within the documentation.